notebooks/203-meter-reader/203-meter-reader-with-paddle-ocr.ipynb

```diff
@@ -0,0 +1,418 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Read img\n",
+    "img = cv2.imread('Example.jpg')\n",
+    "\n",
+    "# Run OCR\n",
+    "results, boxes, txts, scores = run_ocr_models(img, rec_compiled_model, rec_output_layer, det_compiled_model, det_output_layer)\n",
+    "\n",
+    "# Print recognition results\n",
+    "for line in results:\n",
+    "    print(line)\n",
+    "\n",
+    "# Draw text recognition results beside the image.\n",
+    "image = Image.fromarray(cv2.cvtColor(img, cv2.COLOR_BGR2RGB))\n",
+    "draw_img = processing.draw_ocr_box_txt(\n",
+    "    image,\n",
+    "    boxes,\n",
+    "    txts,\n",
+    "    scores,\n",
+    "    drop_score=0.5)\n",
+    "plt.imshow(draw_img)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e5eae497-e670-43f1-bbc1-2d5c850242f6",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Configuration and Pre-processing\n",
+    "Add parameter configuration and make the pre-processing."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a006146a-07c4-4b9d-9f3d-cb2f97a71aba",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Configuration\n",
+    "\n",
+    "Given the area where the reader is in the picture, and config the design size and layout information for the screen."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "af3cb3af-2ff2-4b26-96d8-71560ae6ea93",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "IMG_PATH = 'Example.jpg'\n",
+    "\n",
+    "# The coordinates of the corners of the screen\n",
+    "POINTS = [[1084.5, 64.0],    # Left top\n",
+    "          [3164.5, 160.0],   # right top\n",
+    "          [2978.0, 1791.5],  # right bottom\n",
+    "          [978.5, 1530.5]]   # left bottom\n",
+    "\n",
+    "# The size of the screen\n",
+    "DESIGN_SHAPE = (1300, 1000)\n",
+    "\n",
+    "# Output template\n",
+    "RESULT_TEMP = {\"Info_Probe\":\"探头:---\", \n",
```

Comment:
We need some background introduction for users on what kind of meter in the example image we try to read, for examples: is it for any dedicated purpose and what does it measure? I think we should also add some explanation on the meaning of these parameters in a meter. Since this repo is designed for global users, It could be better if you can explain more on whats the meaning of these Chinese works or just switch the meter image.